docs/source/examples/Widget List.ipynb

```diff
@@ -920,7 +920,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "widgets.Button(\n",
+    "button = widgets.Button(\n",
```

Comment:
Our convention in this file has been to not assign these to variables so that they automatically display. If you feel it is more clear to assign it to a variable, then please add `button` at the bottom of the cell so that when you run the cell, the button is displayed in the output.